demo/screens/__snapshots__/SellOrderClaimScreen.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SellOrderClaimScreen renders the component 1`] = `
+<ContextConsumer>
```

Comment:
This screen's snapshot is so small that the test isn't really testing much. Should we drill down on this screen more?

Reply:
Then again, I guess the presentational components are really where the interesting snapshot tests are, right?

Reply:
This test is small because that is importing `redux` connected HoC. Snapshot is better now after fix. I'll review all tests/snapshots.

Reply:
Makes sense. Sounds good!

Reply:
Done.

Reply:
Thanks! Resolving this